custom_components/powercalc/data/alterco robotics/shelly plug s/model.json

```diff
@@ -0,0 +1,16 @@
+{
+	"measure_description": "Manually measured. Transcribed from https://github.com/bramstroker/homeassistant-powercalc/discussions/207#discussioncomment-1484985",
+	"measure_method": "manual",
+	"measure_device": "Zhirui PR10-D",
+	"name": "Shelly Plug S",
+	"standby_power": 0.52,
+	"standby_power_on": 0.82,
+	"requires_additional_configuration": true,
+	"device_type": "smart_switch",
+	"supported_modes": [
+		"fixed"
+	],
+	"fixed_config": {
+		"watt": 0.82
```

Comment:
This needs to be `power`, not `watt`

Reply:
And you can set the values to `0`. Otherwise the power will be calculated double. 1.64. (power + standby_power_on) when the device is on.

Or you can remove `standby_power_on` and set `power` to 0.82.

Reply:
> you can remove standby_power_on and set power to 0.82.

I chose this one